### PR TITLE
feat(locale): add city names for th

### PR DESCRIPTION
**Pull request by matthewmayer**
Adds city names for `th`. This stops `fakerTH.location.city()` sometimes throwing when #2004 is merged, as `th` had no city patterns, so fell back to en patterns, but sometimes that would select a pattern with a city_prefix, which was empty in `th`.

### CODE REVIEW

File: src/locales/th/location/city.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export default ['{{location.city_name}}'];
```
Comment (Shinigami92) [inline src/locales/th/location/city.ts:1]: Maybe we should first merge the pattern renaming, and then already have this file named city_pattern

Comment (matthewmayer) [reply src/locales/th/location/city.ts:1]: there's no PR for that yet, but can wait for it if you prefer.

Comment (ST-DDT) [reply src/locales/th/location/city.ts:1]: This PR was first so lets merge it first.

**Comment by ST-DDT** [comment]
This needs updating.

**Comment by matthewmayer** [comment]
Test failure seems unrelated see https://github.com/faker-js/faker/issues/2080

**Comment by ST-DDT** [comment]
> Test failure seems unrelated see #2080

Yeah, one of the rare cases where a float() is an int.

**Review by ST-DDT — DISMISSED** [review]
Are the city prefixes/suffixes not applicable to th in general or did you remove them because we dont need them for the current pattern?

**Comment by matthewmayer** [comment]
> Are the city prefixes/suffixes not applicable to th in general or did you remove them because we dont need them for the current pattern?

Both

**Comment by ST-DDT** [comment]
If it is not applicable, then keeping the data as [] shows to the other developers that this is intentionally absent.
Also the [] does not really add big memory or source code overhead, so I think we should keep them.

**Comment by matthewmayer** [comment]
My assumption was that was only needed for definitions which have public methods. Now that citysuffix and cityprefix are no longer methods, seems like they could just be omitted for locales that don't need them?

**Comment by xDivisionByZerox** [comment]
> [...] Now that citysuffix and cityprefix are no longer methods, seems like they could just be omitted for locales that don't need them?

Yes they are indeed no longer accessible via public methods, BUT if the city suffixes/prefixes simply don't exist in the th locale they should be left in. This allows us to track which data is actually missing and which are not applicable. This will become easier after #2078.

**Comment by matthewmayer** [comment]
Sure, restored

**Review by Shinigami92 — DISMISSED** [review]

**Review by Shinigami92 — APPROVED** [review]

**Review by xDivisionByZerox — APPROVED** [review]

**Review by ST-DDT — APPROVED** [review]

**Pull request merged**